src/app/modules/main/chat_section/module.nim

```diff
@@ -77,13 +79,20 @@ proc newModule*(
 
   result.chatContentModules = initOrderedTable[string, chat_content_module.AccessInterface]()
 
+  # Simple community channels uses comminity usersModule while chats uses their own usersModule
+  if isCommunity:
+    result.usersModule = users_module.newModule(
+      events, sectionId, "", true, true,
```

Comment:
Can you add the argument names before the booleans and `""`. It helps understand what is being set

Reply:
done 👍